docs/hyperspectral_rot90.md

```diff
@@ -0,0 +1,35 @@
+## Rotate Hyperspectral Datacubes 
+
+This function rotates a hyperspectral datacube counterclockwise in increments of 90 degrees.  The input and output is a [`Spectral_data` class](Spectral_data.md) 
+instance created while reading in with [`pcv.readimage`](read_image.md) with `mode='envi'`. This function is similar to the [`pcv.rotate`](rotate2.md) functions
+but is specifically suitable for HSI image analysis. 
+
+**plantcv.hyperspectral.rot90**(*spectral_data, k*)
+
+**returns** rot_hsi (instance of the `Spectral_data` class)
+
+- **Parameters:**
+    - spectral_data      - Hyperspectral data instance
+    - k                  - Number of times the array is rotated by 90 degrees
+
+
+- **Example use:**
+    - Below
+    
+    
+
+```python
+
+from plantcv import plantcv as pcv
+
+# Set global debug behavior to None (default), "print" (to file), 
+# or "plot" (Jupyter Notebooks or X11)
+pcv.params.debug = "plot"
+
+# Many plants absorb 430nm wavelength light. 
```

Comment:
Guess I missed this the first time, is this comment also a leftover from a copy-pasted page?